### ISSUE TITLE
UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible.

### ISSUE THREAD

**Issue by Mao-KU** [Tony-Y/pytorch_warmup#5]
Hi Tony,
I am using torch1.9 and ```lr_scheduler.step(lr_scheduler.last_epoch + 1)```, but I got this UserWarning:
UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available.

Will this lead to some learning rate bugs?

**Comment by Tony-Y** [comment]
You should ignore the warning because we need the closed form to use this library (pytorch_warmup).

This comment might be helpful: https://github.com/Tony-Y/pytorch_warmup/pull/1#issuecomment-735620983

**Comment by Mao-KU** [comment]
I see. Thanks for the reply!
